lib/rules/object-property-newline.js

```diff
@@ -40,14 +40,14 @@ module.exports = {
 
         return {
             ObjectExpression: function(node) {
-                var lastTokenOfPreviousValue, firstTokenOfCurrentKey;
+                var lastTokenOfPreviousProperty, firstTokenOfCurrentProperty;
 
                 if (allowSameLine) {
                     if (node.properties.length > 1) {
-                        var firstToken = sourceCode.getFirstToken(node.properties[0].key);
-                        var lastToken = sourceCode.getLastToken(node.properties[node.properties.length - 1].value);
+                        var firstTokenOfFirstProperty = sourceCode.getFirstToken(node.properties[0]);
+                        var lastTokenOfLastProperty = sourceCode.getLastToken(node.properties[node.properties.length - 1]);
```

Comment:
@vitorbal I've also changed these so they would be more verbose and more consistent with the other var names. Is that ok?